server/src/internalClusterTest/java/org/elasticsearch/action/admin/cluster/stats/ClusterStatsIT.java

```diff
@@ -208,7 +208,8 @@ public void testValuesSmokeScreen() throws IOException, ExecutionException, Inte
         assertThat(msg, response.nodesStats.getJvm().getVersions().size(), Matchers.greaterThan(0));
 
         assertThat(msg, response.nodesStats.getVersions().size(), Matchers.greaterThan(0));
-        assertThat(msg, response.nodesStats.getVersions().contains(Version.CURRENT), Matchers.equalTo(true));
+        // TODO: Build.current().unqualifiedVersion() -- or Build.current().version() if/when we move NodeInfo to Build version(s)
+        assertThat(msg, response.nodesStats.getVersions().contains(Version.CURRENT.toString()), Matchers.equalTo(true));
```

Comment:
`getVersions()` is a collection, so you can replace this with

```suggestion
        assertThat(msg, response.nodesStats.getVersions(), hasItem(Version.CURRENT.toString()));
```